### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by sandmaennchen5** [ivaano/NsxLibraryManager#44]
missing DLC ​​updates Tab

**Comment by ZixX3r** [comment]
if you go to the library >Then there are 3 Tabs that say " Title Library| Missing DLC | Missing Updates.

Hope that helps you.

**Comment by sandmaennchen5** [comment]
yes I know that. I meant that for some DLC there are also updates and this display function is missing. for example the projects where it shows the DLC updates

https://github.com/a1ex4/ownfoil

https://github.com/ivaano/NsxLibraryManager

**Comment by ivaano** [comment]
This is already on the latest beta, please check it out.
